pallets/capacity/src/tests/testing_utils.rs

```diff
@@ -71,7 +71,17 @@ pub fn system_run_blocks(n: u32) {
 
 pub fn register_provider(target_id: MessageSourceId, name: String) {
 	let name = Vec::from(name).try_into().expect("error");
-	assert_ok!(Msa::create_registered_provider(target_id.into(), name));
+	let cid = "bafkreidgvpkjawlxz6sffxzwgooowe5yt7i6wsyg236mfoks77nywkptdq"
```

Comment:
Similar to the JS/E2E side, it would probably be useful to have a helper function to create a valid ProviderEntry.